src/webgpu/api/validation/encoding/queries/general.spec.ts

```diff
@@ -3,7 +3,7 @@ TODO:
 
 - Start a pipeline statistics query in all possible encoders:
     - queryIndex {in, out of} range for GPUQuerySet
-    - GPUQuerySet {valid, invalid}
+    - GPUQuerySet {valid, invalid, device mismatched}
     - x ={render pass, compute pass} encoder
```

Comment:
Only items which are not below need to be listed here. Since you added a `.unimplemented()` below, device mismatch doesn't need to be mentioned in this TODO. And if any of this TODO is already done below, it should be removed here too.

Reply:
Never mind I just realized this TODO is for pipeline statistics, but the implementation below is for timestamp.